applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -124,9 +124,10 @@ static void on_event_app_data(const lwm2m_carrier_event_t *event)
 			return;
 		}
 
-		rsp_send("\r\n#XCARRIEREVT: %u,%hhu,\"%s\",%d\r\n", event->type, app_data->type,
```

Comment:
Let's print the data in the second line like TCP and UDP clients

Reply:
I think that it is being done in here. The data is send in the data_send below.

Reply:
Ah, nvm. I see what you mean. There is no line change after the data. Yes, I agree, it would be good for this to work similarly to TCP and UDP

Reply:
Ok, I will move it back

Reply:
I believe I have now aligned it with the format for UDP and TCP.